test-network/network.config

```diff
@@ -0,0 +1,60 @@
+# -i  
+# default image tag
+IMAGETAG="2.5.4"
+# -cai
+# default ca image tag
+CA_IMAGETAG="1.5.7"
+
+# Using crpto vs CA. default is cryptogen
+CRYPTO="cryptogen"
```

Comment:
It would be easier to read if there was a blank line between each property,
especially since some have the flag name and some don't (e.g. `-r` below, does it go with the above property or the next property?).

Reply:
I moved the property to the same line as the description and added a blank line between properties.